website/playground/VersionLink.js

```diff
@@ -16,6 +16,7 @@ export default function({ version }) {
   return ReactDOM.createPortal(
     <a
       href={`https://github.com/prettier/prettier/${href}`}
+      // eslint-disable-next-line react/jsx-no-target-blank
```

Comment:
Could you change `rel="noopener"` below to `rel='noreferrer noopener'` instead of disabling the rule?

Reply:
will do